documentation/source/benchmark.rst

```diff
@@ -0,0 +1,319 @@
+     - 0.080327
+     - 0.082597
+     - 1.285308
+   * - RPCholesky
+     - 0.182340
+     - 0.079254
+     - 0.032427
+     - 0.085621
+     - 1.492112
+   * - SteinThinning
+     - 0.186064
+     - 0.078773
+     - 0.087343
+     - 0.085194
+     - 1.867980
+
+.. list-table:: Coreset Size 50 (Original Sample Size 1_000)
+   :header-rows: 1
+   :widths: 20 15 15 15 15 15
+
+   * - Method
+     - Unweighted_MMD
+     - Unweighted_KSD
+     - Weighted_MMD
+     - Weighted_KSD
+     - Time
+   * - KernelHerding
+     - 0.011488
+     - 0.080717
+     - 0.003478
+     - 0.079936
+     - 2.864534
+   * - RandomSample
+     - 0.083657
+     - 0.084844
+     - 0.004985
+     - 0.079951
+     - 1.237755
+   * - RPCholesky
+     - 0.140705
+     - 0.063904
+     - 0.000732
+     - 0.079960
+     - 1.473465
+   * - SteinThinning
+     - 0.079031
+     - 0.074763
+     - 0.009612
+     - 0.080122
+     - 1.732926
+
+.. list-table:: Coreset Size 100 (Original Sample Size 1_000)
+   :header-rows: 1
+   :widths: 20 15 15 15 15 15
+
+   * - Method
+     - Unweighted_MMD
+     - Unweighted_KSD
+     - Weighted_MMD
+     - Weighted_KSD
+     - Time
+   * - KernelHerding
+     - 0.008475
+     - 0.080237
+     - 0.001907
+     - 0.079933
+     - 2.627561
+   * - RandomSample
+     - 0.032530
+     - 0.077081
+     - 0.001709
+     - 0.080082
+     - 1.257214
+   * - RPCholesky
+     - 0.075108
+     - 0.073218
+     - 0.000880
+     - 0.080010
+     - 1.500886
+   * - SteinThinning
+     - 0.118450
+     - 0.081853
+     - 0.002618
+     - 0.079857
+     - 2.208143
+
+.. list-table:: Coreset Size 200 (Original Sample Size 1_000)
+   :header-rows: 1
+   :widths: 20 15 15 15 15 15
+
+   * - Method
+     - Unweighted_MMD
+     - Unweighted_KSD
+     - Weighted_MMD
+     - Weighted_KSD
+     - Time
+   * - KernelHerding
+     - 0.004305
+     - 0.080290
+     - 0.000598
+     - 0.080049
+     - 2.638131
+   * - RandomSample
+     - 0.048703
+     - 0.077522
+     - 0.000488
+     - 0.080062
+     - 1.265551
+   * - RPCholesky
+     - 0.048309
+     - 0.078012
+     - 0.000846
+     - 0.079993
+     - 1.535722
+   * - SteinThinning
+     - 0.129071
+     - 0.084883
+     - 0.002251
+     - 0.079875
+     - 1.755289
+
+**Visualisation**: The results in this table can be visualised as follows:
+
+  .. image:: ../../examples/benchmarking_images/blobs_benchmark_results.png
+     :alt: Benchmark Results for Synthetic Dataset
+
+  **Figure 3**: Line graphs showing the performance metrics of different coreset
+  algorithms on a synthetic dataset.
+
+Test 3: Benchmarking Coreset Algorithms on Pixel Data from an Image
+-------------------------------------------------------------------
+
+This test evaluates the performance of coreset algorithms on pixel data extracted
+from an input image. The process follows these steps:
+
+1. **Input Image**: An image is loaded and downsampled to reduce resolution (a
+   downsampling factor of 1 corresponds to no downsampling).
```

Comment:
Why mention the downsampling if it's not then used in the test?

Reply:
Removed any reference to downsampling